content/backstage/plugins/github-insights.md

```diff
@@ -19,7 +19,9 @@ coverImageAlt: 'A screenshot of the GitHub Insights plugin. It is showing a code
 gettingStarted: # What will this step accomplish?
   - intro: Install the plugin into Backstage.
     language: bash
-    code: 'yarn add @roadiehq/backstage-plugin-github-insights'
+    code: |
+     // packages/app
+     'yarn add @roadiehq/backstage-plugin-github-insights'
```

Comment:
Thank you for remembering to specify where this commend should be run. There is no need to wrap it in quotes  as far as I know.

```suggestion
     yarn add @roadiehq/backstage-plugin-github-insights
```